druid/src/core.rs

```diff
@@ -995,7 +995,7 @@ mod tests {
             state: &mut state,
         };
 
-        let env = crate::theme::init();
+        let env = crate::Env::default();
```

Comment:
```suggestion
        let env = Env::default();
```
I think `Env` is already in scope, here?